handler.go

```diff
@@ -411,6 +414,7 @@ func (h *Handler) ActionIsPressed(action Action) bool {
 			return true
 		}
 		if h.keyIsPressed(k) {
+			h.last = k.kind.device()
```

Comment:
Should `Info*` methods be updated as well? It's OK to copy/paste this assignment everywhere for now, and then maybe we can refactor this stuff to somehow

Reply:
I had a single-line method for this, but decided to renove it. Probably worth returning.

Reply:
I'm mostly talking about `JustPressedActionInfo` methods that don't call `ActionIsPressed` method internally; or maybe I missed them updating the last device field in the code

Reply:
Moved duplication assignments to a method in the end.